src/sections/create-dataset/CreateDatasetForm.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { Alert, Button, Col, Form, Row } from '@iqss/dataverse-design-system'
+import { useTranslation } from 'react-i18next'
+import { RequiredFieldText } from '../../components/form/RequiredFieldText/RequiredFieldText'
+import { SeparationLine } from '../../components/layout/SeparationLine/SeparationLine'
+import { useCreateDatasetForm, SubmissionStatusEnums } from './CreateDatasetContext'
+import {
+  FormValidationService,
+  FormSubmissionService
+} from '../../dataset/domain/useCases/createDataset'
+import styles from '/src/sections/dataset/Dataset.module.scss'
+interface FormPresenterProps {
+  formValidationService: FormValidationService
+  formSubmissionService: FormSubmissionService
+}
+export const CreateDatasetFormPresenter: React.FC<FormPresenterProps> = ({
+  formValidationService,
+  formSubmissionService
+}: FormPresenterProps) => {
+  const {
+    formErrors,
+    submissionStatus,
+    handleCreateDatasetFieldChange,
+    handleCreateDatasetSubmit
+  } = useCreateDatasetForm({ formValidationService, formSubmissionService })
+
+  const { t } = useTranslation('createDataset')
+
+  return (
+    <>
+      <article>
+        <header className={styles.header}>
+          <h1>{t('pageTitle')}</h1>
+        </header>
+        <SeparationLine />
+        <div className={styles.container}>
+          <RequiredFieldText />
+          {submissionStatus === SubmissionStatusEnums.IsSubmitting && <p>Submitting...</p>}
```

Comment:
The text inside the <p> should come from the translations file `<p>Submitting...</p> ` =>`<p>{t('submitting')}</p>`

The same applies for the other paragraphs